dogs.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Adopt a Dog</title>
+  </head>
+  <style media="screen">
+
+    header {
+      background: grey;
+      width: 100%;
+      height: 15%;
+    }
+
+    footer {
+      background: grey;
+      width: 100%;
+      height: 15%;
+    }
+
+    .cart {
+      width: 2rem;
+      position: absolute;
+      top: 5%;
+      right: 10%;
+      color: black;
+    }
+
+    .checkout-button {
+      width: 2rem;
+      position: absolute;
+      top: 10%;
+      right: 10%;
+      color: black;
+
+    }
+
+    .logo {
+      width: 30%;
+    }
+
+    .container {
+      display: flexbox;
+      width: 25%;
+      float: left;
+
+    }
+
+  </style>
+  <body>
+    <header>
+      <img src="images/adoptadog-logo.svg" class="logo" alt="Dog sitting next to AdoptaDog text">
+      <p>Seattle's premiere dog adoption agency</p>
+      <img src="images/cart-icon.svg" class="cart" alt="shopping cart">
+      <button class=checkout-button><a href="checkout.html">checkout</a></button>
+    </header>
+    <nav>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="dogs.html">Dogs</a></li>
+        <li><a href="blog.html">Blog</a></li>
+      </ul>
+    </nav>
+    <section>
+      <h1>Here are all of our dogs!</h1>
+    </section>
+    <main>
+      <div class="container">
+        <img src="images/murphy-card.jpg" alt="">
+        <h2>Murphy</h2>
+        <h3>Cost to Adopt: $123.45</h3>
+        <p>
+          dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+          Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit
+          anim id est laborum.
+        </p>
+        <button type="button" name="button">Adopt</button>
+      </div>
+      <div class="container">
+        <img src="images/poppy-card.jpg" alt="">
+        <h2>Poppy</h2>
+        <h3>Cost to Adopt: $123.45</h3>
+        <p>
+          dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+          Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit
+          anim id est laborum.
+        </p>
+        <button type="button" name="button">Adopt</button>
+      </div>
+      <div class="container">
+        <img src="images/jack-card.jpg" alt="">
+        <h2>Jack</h2>
+        <h3>Cost to Adopt: $123.45</h3>
+        <p>
+          dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+          Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit
+          anim id est laborum.
+        </p>
+        <button type="button" name="button">Adopt</button>
+      </div>
+      <div class="container">
```

Comment:
container is a pretty generic term. It works, but maybe think of a more descriptive class name? dog-container is an idea, something like that.